app/helpers/token.js

```diff
@@ -0,0 +1,13 @@
+const jsrasign = require('jsrsasign'),
+  algorithm = 'HS256',
+  tokenType = 'JWT',
+  pass = 'c2VjcmV0';
```

Comment:
This should be configurable by `.env`. Else if someone gets access to the source code they can manipulate the tokens for malicious purposes.